client/src/components/Authentication/Login.jsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import Styled from 'styled-components';
+
+// Styles
+
+// Main wrapper
+const Wrapper = Styled.div`
+    width: 30%;
+    margin: 80px auto;
+    padding-left: 3%;
+    color: #fff;
+
+    input {
+        width: 400px;
+        margin-bottom: 20px;
+        height: 40px;
+    }
+
+    textarea {
+        height: 500px;
+    }
+    
+    h3 {
+        font-size: 16px;
+        color: #E7A837;
+    }
+    
+`;
+
+// Buttons
+const Button = Styled.button`
+    width: 160px;
+    height: 40px;
+    cursor: pointer;
+    font-size: 16px;
+    border: 1px solid #E7A837;
+    border-radius: 20px;
+    background-color: #E7A837;
+    color: #000;
+    
+    &:hover {
+        background-color: #fff;
+    }
+`;
+
+// Error to display on form
+const Error = Styled.p`
+    color: red;
+`;
+
+// Paragraph
+const P = Styled.p`
+    color: #4a494a;
+    font-size: 20px;
+    margin-left: 3%;
+`
+
+/**
+ * Render the Login
+ * @param props - Props
+ * @returns {*}
+ * @constructor
+ */
+function Login(props) {
+
+    /**
+     * Logs the user in
+     * @param e - Event
+     */
+    function handleSubmit(e) {
+        e.preventDefault();
+        props.login();
+    }
+
+    /**
+     * Sends the user to the game endpoint after successful login
+     */
+    function sendGame() {
```

Comment:
I like how this is a function, especially since you can reuse it for both logging in and registering.